docs/output.md

```diff
@@ -269,6 +270,35 @@ Protein-coding genes are predicted for each assembly.
 
 </details>
 
+## Virus identification in assemblies
+
+### geNomad
+
+[geNomad](https://github.com/apcamargo/genomad) identifies viruses and plasmids in sequencing data (isolates, metagenomes, and metatranscriptomes)
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `VirusIdentification/geNomad/[sample/group]/`
+  - `[sample/group]_annotate`
+    - `[sample/group]_taxonomy.tsv`: Taxonomic assignment data
+  - `[sample/group]_aggregated_classification`
+    - `[sample/group]_aggregated_classification.tsv`: Sequence classification in tabular format
+  - `[sample/group]_find_proviruses`
+    - `[sample/group]_provirus.tsv`: Characteristics of proviruses identified by geNomad
+  - `[sample/group]_summary`
+    - `[sample/group]_virus_summary.tsv`: Virus classification summary file in tabular format
+    - `[sample/group]_plasmid_summary.tsv`: Plasmid classification summary file in tabular format
+    - `[sample/group]_viruses_genes.tsv`: Virus gene annotation data in tabular format
+    - `[sample/group]_plasmids_genes.tsv`: Plasmid gene annotation data in tabular format
+    - `[sample/group]_viruses.fna`: Virus nucleotide sequences in FASTA format
+    - `[sample/group]_plasmids.fna`: Plasmid nucleotide sequences in FASTA format
+    - `[sample/group]_viruses_proteins.faa`: Virus protein sequences in FASTA format
+    - `[sample/group]_plasmids_proteins.faa`: Plasmid protein sequences in FASTA format
+  - `[sample/group].log`: Plain text log file detailing the steps executed by geNomad (annotate, find-proviruses, marker-classification, nn-classification, aggregated-classification and summary)
```

Comment:
Do the names of the files purely have sample/group? Without looking at the rest of the code yet - are you sure these don't need to be output on a per assember+binner+sample basis?

Reply:
geNomad's output files are named based on the input file (i.e. input = test1.fasta; outputs = test1_virus_summary.tsv). If you think it is necessary to name outputs based on assembler+binner+sample, I could make a patch to the geNomad module where the output files are all renamed!

Reply:
Poking about - modules.conf doesn't specify `ext.prefix`, so the output files should just be named `[sample/group]_*.*` - might run into overwriting of results if more than one assembler runs. Should fix by specifying `ext.prefix = { "${meta.id}-${meta.assembler}" }` or similar in the config file. This runs before binning so there should be no need to specify the binner too.

Reply:
@prototaxites I think though was the purpose of this comment

> geNomad's output files are named based on the input file

So there is no way to specify the output explicitly? Therefore I _assume_ that as the input file _should_ have the assembly/binner name, then we don't need to use `prefix` and manual `mv` commands in the module. However this is somethign for @CarsonJM to confirm

Reply:
Ah, good point - I was thrown because `prefix` is defined in the module, but it turns out it's not actually used within the script block. In that case, I agree we shouldn't see any conflicts - but perhaps worth updating the docs to reflect the output file names?

Reply:
Agreed! @CarsonJM please double check and update as necessary :pray:

Reply:
The current output looks like this:
`MEGAHIT-group-0.contigs_summary`

So should the output docs say something like:
`VirusIdentification/geNomad/[assembler]-[sample/group]_summary`

The only part I'm unsure about is that the `.contigs` would be `_scaffolds` if metaSPAdes is chosen as the assembler

Reply:
I think you could just throw a wildcard `[assembler]-[sample/group]_*_summary` in there to catch the difference between `.contigs` and `_scaffolds`. Not a perfect solution but I think it's clear enough!